aws/cloudformation/moz-single.json

```diff
@@ -487,7 +487,37 @@
           }
         ]
       }
+    },
+
+    "FxaCloudfrontPolicy" : {
+      "Type" : "AWS::CloudFront::Distribution",
+      "Properties" : {
+          "DistributionConfig" : {
+              "DefaultCacheBehavior" : {
+                  "AllowedMethods" : [ "GET", "HEAD", "OPTIONS" ],
+                  "TargetOriginId" : "FxaCloudfrontCDN",
+                  "ForwardedValues" : {
+                      "QueryString" : "false",
+                      "Cookies" : { "Forward" : "none" }
+                  },
+                  "ViewerProtocolPolicy" : "allow-all"
+              },
+              "Enabled" : "true",
+              "Origins" : [ {
+                  "DomainName" : { "Fn::Join" : [ "", [{"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}, "."]]},
```

Comment:
Yeah, we don't want the final dot there. We do for Route 53, but not here.

It ought to be:

```json
"DomainName" : { "Fn::Join" : [ "", [{"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}]]
```

The example here doesn't have a final dot:
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/quickref-cloudfront.html